zzk/service/service.go

```diff
@@ -267,6 +267,7 @@ func (l *ServiceListener) sync(svc *service.Service, rss []dao.RunningService) b
 		// resumeInstance updates the service state ONLY if it has a PAUSED DesiredState
 		if err := resumeInstance(l.conn, state.HostID, state.ID); err != nil {
 			glog.Warningf("Could not resume paused service instance %s (%s) for service %s on host %s: %s", state.ID, state.Name, state.ServiceID, state.HostID, err)
+			return false
```

Comment:
I don't know enough about this method to know if the following suggestion is a good idea or not, but we're in a loop here, so maybe we should try to resume all of the instances, and return false after the loop if one or more could not be resumed?